src/queuemailerl_event.erl

```diff
@@ -24,7 +36,9 @@ parse(RawEvent) ->
     of
         Event -> {ok, Event}
     catch
-        error:Reason -> {error, Reason}
+        error:Reason ->
+            io:format("Error ~p~nTraceback: ~p", [Reason, erlang:get_stacktrace()]),
```

Comment:
Vi kanske bör dra in lager eller alternativt använda error_logger här istället. Vet inte hur io:format/2 hanteras i loggning till fil m.m.